src/components/NavMenuItemBase.vue

```diff
@@ -70,32 +70,5 @@ export default {
       transition-delay: $nav-menu-item-stagger-delay;
     }
   }
-  // create a staggered effect, only each settings element (language toggle, API changes)
```

Comment:
Not sure if it's caused by this, but I noticed that the animation of the nav dropdown closing in the small viewport seems to have regressed. Now it fades and disappears and does not fade up. 

https://github.com/apple/swift-docc-render/assets/87735557/78073e56-b267-4df0-bdeb-03c03e20ecd6